balena/models/device.py

```diff
@@ -1592,3 +1592,34 @@ def get_mac_address(self, uuid):
             return device['mac_address'].split()
         else:
             return []
+
+    def get_metrics(self, uuid):
+        """
+        Get the metrics related information for a device.
+
+        Args:
+            uuid (str): device uuid.
+
+        Returns:
+            dict: metrics of the device.
+
+        Raises:
+            DeviceNotFound: if device couldn't be found.
+
+        """
+
+        device = self.get(uuid)
+        if "memory_usage" in device:
+            return {
+                "memory_usage": device["memory_usage"],
```

Comment:
can we make a list of all fields we want and extract it like this?
```
metrics = ['memory_usage', 'memory_total']
return { k: device.get(k, '') for k in metrics }
```

It will be a bit nicer and we don't have to repeat all the field